app/workers/send_text_message_worker.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class SendTextMessageWorker
+  include Sidekiq::Worker
+
+  def perform(number, message)
+    $twilio.api.account.messages.create(from: '+16309841220', to: number, body: message) if number
```

Comment:
`from` phone number should be a constant that's accessible from whatever classes need access to it.

Reply:
Ended up making this app config: `Rails.configuration.twilio_number`